demos/top-app-bar.html

```diff
@@ -203,7 +208,17 @@ <h3>Demo Controls</h3>
           if (!this.checked) {
             appBarEl.classList.remove('mdc-top-app-bar--short-collapsed');
             appBarEl.classList.remove('mdc-top-app-bar--short-has-action-item');
+            alwaysClosedCheckbox.checked = false;
           }
+
+          alwaysClosedCheckbox.disabled = !this.checked;
+        });
+
+        alwaysClosedCheckbox.addEventListener('change', function() {
+          this.checked ? appBarEl.classList.add('mdc-top-app-bar--short-collapsed') : appBarEl.classList.remove('mdc-top-app-bar--short-collapsed');
```

Comment:
Same comment as last time, move the ternary to within square brackets like what was done for shortCheckbox in the previous PR